readyplayerme/rpm-unity-sdk-avatar-creator
# [SDK-52] Set selected icon to avatar prediction assets

## [SDK-52](https://ready-player-me.atlassian.net/browse/SDK-52)

## Description

- Added setting of selected button icon to avatar's selected assets when using default avatars or profile avatars

## Changes

#### Added

- Set selected icon to avatar prediction assets

#### Updated

- Package version
- Readme
- Change AvatarCreatorData menu path

## How to Test

-  Choose a default avatar or a user avatar and observe assets selected icon in asset different panels



## Checklist


## QA Testing

-   Mention any useful test cases for this feature and add to QA Testing documentation.

## Details

-   If more details are necessary to support the description with images and videos add them here.







[SDK-52]: https://ready-player-me.atlassian.net/browse/SDK-52?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
